crates/re_ui/src/lib.rs

```diff
@@ -531,6 +536,44 @@ impl ReUi {
         });
     }
 
+    /// Paint a collapsing triangle with rounded corners.
+    ///
+    /// Alternative to [`egui::collapsing_header::paint_default_icon`].
+    pub fn paint_collapsing_triangle(ui: &mut egui::Ui, openness: f32, response: &egui::Response) {
+        let visuals = ui.style().interact(response);
+
+        let rect = response.rect;
+        let extent = response.rect.width().min(response.rect.height());
+
+        // normalised in [0, 1]^2 space
+        let mut points = vec![
+            pos2(0.80387, 0.470537),
+            pos2(0.816074, 0.5),
+            pos2(0.80387, 0.529463),
+            pos2(0.316248, 1.017085),
+            pos2(0.286141, 1.029362),
+            pos2(0.257726, 1.017592),
+            pos2(0.245118, 0.987622),
+            pos2(0.245118, 0.012378),
+            pos2(0.257726, -0.017592),
+            pos2(0.286141, -0.029362),
+            pos2(0.316248, -0.017085),
+            pos2(0.80387, 0.470537),
+        ];
```

Comment:
Perhaps link to https://github.com/rerun-io/rerun/issues/2960 here so we remember to update this code once that issue is resolved